server/runtime/src/test/java/org/infinispan/server/test/ContainerInfinispanServerDriver.java

```diff
@@ -134,14 +137,14 @@ protected void start(String name, File rootDir, String configurationFile) {
                               9999   // JMX Remoting
                         )
                         .build());
-      latch = new CountdownLatchLoggingConsumer(configuration.numServers(), ".*ISPN080001.*");
+      latch = new CountdownLatchLoggingConsumer(configuration.numServers(), INFINISPAN_SERVER_STARTUP_REGEX);
       for (int i = 0; i < configuration.numServers(); i++) {
          GenericContainer container = createContainer(i, rootDir);
          containers.add(i, container);
          log.infof("Starting container %s-%d", name, i);
          container.start();
       }
-      Exceptions.unchecked(() -> latch.await(10, TimeUnit.SECONDS));
+      Exceptions.unchecked(() -> latch.await(serverStartupTimeout, TimeUnit.SECONDS));
```

Comment:
The test setup should be throwing an exception if the server didn't start within the timeout, but `CountDownLatch.await()` doesn't do anything if the timeout expires.